NEWS.md

```diff
@@ -2,6 +2,14 @@
 
 ### Changed
 
+* rename `select_causal_rules()` function in `lasso_rules_filter()` and 
```

Comment:
Please focus more on the current state of the code. For example, in this case, you can say: `select_causal_rules()` is now `lasso_rules_filter().`